archetypes/Exchange/Summary/index.tsx

```diff
@@ -130,7 +139,7 @@ export default (({ pool, showBreakdown, amount, isLong, commitAction, receiveIn
                         <>
                             <Section label="Expected Token Value">
                                 <SumText>
-                                    {`${toApproxCurrency(calcNotionalValue(tokenPrice, amount), 3)}`} USDC
+                                    {`${toApproxCurrency(calcNotionalValue(tokenPrice, amount), 2)}`} USDC
```

Comment:
since some markets are Frax, you may have to use the other side of the `selectedToken` split to get the collateral, alternatively use `pool.quoteToken.symbol` since it will be more accurate